apps/api/src/app/workflows-v2/dto/create-workflow.response.dto.ts

```diff
@@ -0,0 +1,3 @@
+export class CreateWorkflowResponseDto {
+  // TBD
```

Comment:
todo: create response dto based on the framework workflow define interface (v2 workflow).